google/cloud/bigquery/v2/minimal/internal/job.h

```diff
@@ -0,0 +1,60 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_BIGQUERY_V2_MINIMAL_INTERNAL_JOB_H
+#define GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_BIGQUERY_V2_MINIMAL_INTERNAL_JOB_H
+
+#include "google/cloud/version.h"
+
+namespace google {
+namespace cloud {
+namespace bigquery_v2_minimal_internal {
```

Comment:
Given that the feature now seems to be branded as "bigquery_rest", should we be updating path names and namespaces?

Reply:
I don't think so. In my mind `bigquery_rest` is all encompassing and could include some `google::cloud::bigquery_v2_foo_internal` at a later date.

My understanding of this is limited though. (Which implies I probably did not have enough foresight to justify changing the name of the feature. Oh well.)

Reply:
OK.